ipaserver/install/dogtaginstance.py

```diff
@@ -171,6 +167,15 @@ def spawn_instance(self, cfg_file, nolog_list=()):
         except ipautil.CalledProcessError as e:
             self.handle_setup_error(e)
 
+    def _clean_pkispawn_files(self):
+        if self.tmp_agent_db is None:
```

Comment:
Right, it was my impression that this one would not exist if `pkispawn`was not run, I can just remove this check and run removal of both trees.